governance-aepp/src/components/AeIdentityLight.vue

```diff
@@ -215,66 +178,50 @@
     }
 
     .currency-symbol {
-      letter-spacing: 0.12em;
-      padding-right: 3px;
+      letter-spacing: 0.12rem;
+      padding-right: 0.1875rem;
     }
 
     .ae-identity-light .avatar {
-      border-width: 1px;
-      width: 34px;
-      height: 34px;
+      border-width: 0.0625rem;
+      width: 2.125rem;
```

Comment:
i think its not that we forcefully need to remove px everywhere, it can make sense in some places